tests/kernels/moe/test_cpu_fused_moe.py

```
@@ -589,3 +589,115 @@ def test_cpu_fused_moe_unaligned_intermediate_size(
+    dtype = torch.bfloat16
+    up_dim = 2 * intermediate_size
+
+    layer = _StubMoELayer(
+        torch.randn((expert_num, up_dim, hidden_size), dtype=dtype),
+        torch.randn((expert_num, hidden_size, intermediate_size), dtype=dtype),
+        MoEActivation.SWIGLUOAI,
+    )
+
+    if torch.cpu._is_amx_tile_supported():
+        with pytest.raises(RuntimeError):
+            CPUFusedMOE(layer)
```

Ruff (0.16.3)

[error] Undefined name `CPUFusedMOE`

(F821)